packages/storage/amplify_storage_s3/example/integration_test/list_test.dart

```diff
@@ -20,32 +20,61 @@ void main() {
       '$uniquePrefix/file2.txt',
       '$uniquePrefix/subdir/file3.txt',
       '$uniquePrefix/subdir2#file4.txt',
+      '$uniquePrefix/file5.txt',
+      '$uniquePrefix/file6.txt',
+      '$uniquePrefix/subdir3/file7.txt',
+      '$uniquePrefix/subdir4#file8.txt',
     ];
     group('standard config', () {
+      final mainBucket =
+          StorageBucket.fromOutputs('Storage Integ Test main bucket');
+      final secondaryBucket = StorageBucket.fromOutputs(
+        'Storage Integ Test secondary bucket',
+      );
       setUpAll(() async {
         await configure(amplifyEnvironments['main']!);
-
-        for (final path in uploadedPaths) {
+        for (var i = 0; i < 4; i++) {
```

Comment:
This was changed because half the uploaded paths should go to one bucket, and the other half to another. I changed the loop to use descriptive variable names now instead of just the hard coded list lengths